client/blocks/plan-thank-you-card/style.scss

```diff
@@ -0,0 +1,226 @@
+.plan-thank-you-card {
+	position: relative;
+	width: 100%;
+	max-width: 500px;
+	background-color: darken( $alert-green, 10 );
+	border-radius: 8px 8px 6px 6px;
+	overflow: hidden;
+	margin: 56px auto 24px;
+	color: $white;
+	font-family: $sans;
+	text-align: center;
+
+	@media ( max-width: 500px ) {
```

Comment:
Do we really need to use this custom media query? We have some defined in `assets/stylesheets/shared/mixins/_breakpoints` and should add custom ones only if reaaally needed. :) See https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md#the-dry-principle.